jsapp/js/components/common/button.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import Button from './button';
+
+import {render, screen} from '@testing-library/react';
+import {describe, it, expect, jest} from '@jest/globals';
+import userEvent from '@testing-library/user-event';
+
+describe('Button', () => {
+  it('Should render and be clickable when enabled', async () => {
+    const user = userEvent.setup();
+
+    // Mock
+    const handleClickFunction = jest.fn();
+
+    // Render
+    render(
+      <Button
+        type={'primary'}
+        size={'l'}
+        label='Button Label'
```

Comment:
Do we have a linter rule to convert `{'l'}` into `'l'`?

Reply:
It was my mistake to leave it like that. Fixed now!